### ISSUE TITLE
Output.dat is not created

### ISSUE THREAD

**Issue by HamidArianZad** [issp-center-dev/TeNeS#67]
Dear TeNeS team,

In my recent TN simulations using TeNeS, I encountered an issue with saving the calculation results in the 'output.dat' file. I'm currently using the command $ python tutorial_magnetization.py > output.dat to save the output. However, despite the file being created, the results are not written into it. Consequently, the calculation process halts, and when I check the 'output.dat' file, I only see the following lines.
```
Number of Processes: 1
Number of Threads / Process: 16
Tensor type: real
Bond dimensions:
   D  (Bulk): 2
   chi (CTM): 10
Start simple update
```
after line "Start simple update" there is nothing. 
How can I write the results into the output.dat during the calculation time?

**Comment by yomichi** [comment]
- From the last successful run, did you change something?
    - python script, simulation parameter, tenes itself (version, branch, commit), environments (e.g., OS)
- Could you give more information?
    - version of TeNeS
    - input files
        - `tutorial_magnetization.py`
        - `input.toml` which fails

**Comment by HamidArianZad** [comment]
In fact, I previously used TeNeS in my previous notebook, and everything worked fine. I have now installed TeNeS on my new notebook and attempted to save the results of my calculations into the 'output.dat' file as usual. However, I encountered some difficulties.

The version of TeNeS installed on my notebook is 1.3.1: 2022-10-21. The file 'tutorial_magnetization.py' can be found in the folder '/TeNeS/sample/05_magnetization', which was automatically created during the TeNeS installation process.

The input file, 'input_0_1500.toml', was generated by running the 'tutorial_magnetization.py' file.

I have not made any changes to the parameters in the files located in the '/TeNeS/sample/05_magnetization' folder. I simply installed TeNeS on my new notebook and executed the command '$ python tutorial_magnetization.py > output.dat'.

**Comment by yomichi** [comment]
In your new notebook, does TeNeS v1.2.0 work (`sample/05_magnetization`)?

**Comment by HamidArianZad** [comment]
I have not attempted it.

Should I uninstall the current version, 1.3.1: 2022-10-21, and install the older version, v1.2.0? 
If necessary, I would prefer to continue working with the newer version, 1.3.1: 2022-10-21, as I believe it performs better than the older version. 
In the newer version (ver: 1.3.1), if I execute the command `$ python tutorial_magnetization.py` without appending `> output.dat`, the program runs successfully until the completion of the calculations. I can then access the results of the magnetization, which are saved in the file `magnetization.dat`. Other quantities are also saved after the calculations are finished.

I think the results for the local magnetization in Heisenberg spin models are more precise with the newer version v1.3.1 compared to older version v1.2.0.

**Comment by yomichi** [comment]
Yes, the newer version is recommended, of course (FYI, we released v1.3.2 yesterday).

> In the newer version (ver: 1.3.1), if I execute the command $ python tutorial_magnetization.py without appending > output.dat, the program runs successfully until the completion of the calculations. I can then access the results of the magnetization, which are saved in the file magnetization.dat. Other quantities are also saved after the calculations are finished.

Is the problem solved?

**Comment by HamidArianZad** [comment]
Appending `> output.dat` in version `ver: 1.3.1` did not resolve the issue for me.

I am pleased to hear about the release of the latest version, `v1.3.2`. I would like to upgrade to this version. Could you please provide me with the command to upgrade from `v1.3.1` to `v1.3.2`? 
Alternatively, do I need to uninstall the older version, `v1.3.1`, and then install the new version, `v1.3.2`?
